The linker flag needs to be adjusted in order to find OpenSSL 3.

Not intended for upstream.

--- pycurl-7.44.1/setup.py
+++ pycurl-7.44.1/setup.py
@@ -229,7 +229,7 @@ class ExtensionConfiguration(object):
         OPENSSL_DIR = scan_argv(self.argv, "--openssl-dir=")
         if OPENSSL_DIR is not None:
             self.include_dirs.append(os.path.join(OPENSSL_DIR, "include"))
-            self.library_dirs.append(os.path.join(OPENSSL_DIR, "lib"))
+            self.library_dirs.append(os.path.join(OPENSSL_DIR, "lib/64"))
         try:
             p = subprocess.Popen((self.curl_config(), '--version'),
                 stdout=subprocess.PIPE, stderr=subprocess.PIPE)
